### PR TITLE
OE-6673 - Virtual Clinic review panel vertical height takes up too much screen real estate

### PR DESCRIPTION
**Pull request by edgaaar8**

### CODE REVIEW

**Comment by biskyt** [comment]
Tested and passed on JIRA - Code review needed + Conflict in style.css to resolve before merge @splatEric, would you be so kind to review / merge as soon as time permits!

**Comment by splatEric** [comment]
We dealt with this one right? So we can just close this out?

**Comment by biskyt** [comment]
Tested, these changes were not already merged in to develop and still require merging....

File: protected/assets/sass/openeyes/components/_alert-boxes.scss [hunk 1]
```diff
@@ -65,4 +65,5 @@
 @import "alert-boxes/warning";
 @import "alert-boxes/error";
 @import "alert-boxes/watermark";
-@import "alert-boxes/injection-warning";
\ No newline at end of file
+@import "alert-boxes/injection-warning";
+@import "alert-boxes/patientticketing";
```
Comment (splatEric) [inline protected/assets/sass/openeyes/components/_alert-boxes.scss]: css for a module should not be part of the core code

File: protected/assets/sass/openeyes/components/alert-boxes/_patientticketing.scss [hunk 2]
```diff
@@ -0,0 +1,47 @@
+
```
Comment (splatEric) [inline protected/assets/sass/openeyes/components/alert-boxes/_patientticketing.scss]: This should be part of patient ticketing module, not the core

File: protected/components/BaseAPI.php [hunk 3]
```diff
@@ -217,7 +217,7 @@ public function getLatestElement($element, Patient $patient, $use_context = fals
      */
     public function getElementForLatestEventInEpisode($episode, $element)
     {
-        trigger_error('getElementForLatestEventInEpisode is deprecated as of version 2.0, please use getElementFromLatestEvent instead', E_USER_NOTICE);
+        //trigger_error('getElementForLatestEventInEpisode is deprecated as of version 2.0, please use getElementFromLatestEvent instead', E_USER_NOTICE);
```
Comment (splatEric) [inline protected/components/BaseAPI.php]: this should not be committed

Comment (neilsf) [inline protected/components/BaseAPI.php]: Withdrawn. Please comment this line on your local instance when testing.

**Comment by splatEric** [comment]
I'll have to look at this one next week

**Comment by edgaaar8** [comment]
fixed and re-tested, please review

**Comment by biskyt** [comment]
Getting the following error again when trying to add a patient to VC:

```
getElementForLatestEventInEpisode is deprecated as of version 2.0, please use getElementFromLatestEvent instead

/var/www/openeyes/protected/components/BaseAPI.php(220)

208 
209     /**
210      * gets the element of type $element for the given patient in the given episode.
211      *
212      * @param Episode $episode - the episode
213      * @param string  $element - the element class
214      *
215      * @return BaseEventTypeElement|null - Will actually be an instance of the class requested
216      * @deprecated since 2.0
217      */
218     public function getElementForLatestEventInEpisode($episode, $element)
219     {
220         trigger_error('getElementForLatestEventInEpisode is deprecated as of version 2.0, please use getElementFromLatestEvent instead', E_USER_NOTICE);
221         $event_type = $this->getEventType();
222 
223         if ($event = $episode->getMostRecentEventByType($event_type->id)) {
224             $criteria = new CDbCriteria();
225             $criteria->compare('episode_id', $episode->id);
226             $criteria->compare('event_id', $event->id);
227             $criteria->order = 'event.created_date desc';
228 
229             return $element::model()
230                 ->with('event')
231                 ->find($criteria);
232         }
```

**Comment by splatEric** [comment]
This might have issues with OE-6753 that I just merged into develop, can someone please do another merge and verify the functionality?

**Comment by biskyt** [comment]
all known issues now fixed. conflicts need resolving before merge!

**Pull request merged**